auspice-client/customisations/navbar.js

```diff
@@ -123,8 +124,8 @@ const NavBar = ({sidebar, narrativeTitle, width}) => {
           null : (
             <div style={{...styles.flexColumns, paddingRight: "12px"}}>
               <div style={{flex: 5}}/>
-              {renderLink("DOCS", "/docs", styles.link)}
-              {renderLink("HELP", "/help", styles.link)}
+              <Link href="https://docs.nextstrain.org/en/latest/index.html" style={styles.link} target="_blank" rel="noopener noreferrer" text={<React.Fragment>DOCS <FaExternalLinkAlt/></React.Fragment>}/>
```

Comment:
These links to a different subdomain of nextstrain.org are best thought of as [internal links](https://www.seroundtable.com/subdomains-google-webmaster-tools-13960.html) and thus there should be no change in how they're rendered (the addresses themselves do need changing as you've done here).

Reply:
Ah, thanks -  I didn't know that! Considering we are treating them as internal links, I think staying in the same tab is more in line with that. As I was considering them external, i figured a separate tab would be appreciated by users but that is also based mostly in my own preference. 

To be clear, when such links are rendered in the static-site we still want to move away from them being gatsby-server-handled links right?

Reply:
> when such links are rendered in the static-site we still want to move away from them being gatsby-server-handled links right?

Yes -- I believe those are only used for links which remain internal to the Gatsby-produced section of the site